src/rviz_visual_tools.cpp

```diff
@@ -1077,6 +1077,58 @@ bool RvizVisualTools::publishPolygon(const geometry_msgs::Polygon &polygon, cons
   publishPath(points, color, scale, ns);
 }
 
+  bool RvizVisualTools::publishWireframeCuboid(const Eigen::Affine3d &pose,
+					       const Eigen::Vector3d &position,
+					       const Eigen::Matrix3d &rotation_matrix,
+					       const Eigen::Vector3d &min_point,
+					       const Eigen::Vector3d &max_point,
+					       const RvizVisualTools::colors &color=RvizVisualTools::BLUE)
```

Comment:
I actually wanted to talk to you about this tomorrow, because I agree that pose is redundant for a general purpose cuboid drawing, but that's what we need to draw the pcl-computed bounding boxes at an arbitrary mesh pose. OTOH, maybe it's possible to take pcl's rotation matrix and position, put them together in an affine transform, and call `publishWireframeCuboid`with `pose * (rotation + position)`?